src/EventSubscriber/PageNotFoundEventSubscriber.php

```diff
@@ -24,7 +24,7 @@
 use Drupal\Core\Path\PathValidatorInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
```

Comment:
@shishir-intelli Do you believe that we need to update the module's Drupal version to 9, to accept this change?

Reply:
@phdhiren 
Yes, This method has been deprecated since Symfony 4.4